docs/snippets/gettingStarted/webApi/simpleApi.ts

```diff
@@ -139,6 +141,40 @@ export const shoppingCartApi =
         return NoContent();
       }),
     );
+
```

Comment:
**FIX:** @thiagomini, for now I'd prefer to keep the sample outside of the `complete-api` region as the example in the getting started is (for now) focused on business logic and writing. I'm planning to introduce the read model side as the next step. Would you mind moving this endpoint outside the region? 

The method as a test method is okay as it is. You could consider providing the customised evolve method that would already build the desired read model without the need to do the additional mapping. But that's a nice to have if you wanted to play more with event sourcing. This would be a dedicated `ShoppingCartDetails` type with is own `evolve` function.